backend/src/connectors/store.py

```
@@ -249,9 +281,19 @@ def flush(reason="batch"):
         flush("final")
 
     # Update status manifest
-    manifest.add_completed_source(source.name)
+    manifest.add_completed_source(source)
+    if hasattr(manifest, "set_config_hash"):
+        try:
+            manifest.set_config_hash(current_config_hash)
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)